entity/shaders/gaussian_blur.frag

```diff
@@ -16,19 +16,19 @@ in vec2 v_texture_coords;
 in vec2 v_src_texture_coords;
 in vec2 v_texture_size;
 in vec2 v_blur_direction;
+in float v_blur_sigma;
 in float v_blur_radius;
 in float v_src_factor;
 in float v_inner_blur_factor;
 in float v_outer_blur_factor;
 
 out vec4 frag_color;
 
-const float kTwoPi = 6.283185307179586;
+const float kSqrtTwoPi = 2.50662827463;
 
 float Gaussian(float x) {
-  float stddev = v_blur_radius * 0.5;
-  float xnorm = x / stddev;
-  return exp(-0.5 * xnorm * xnorm) / (kTwoPi * stddev * stddev);
+  float variance = v_blur_sigma * v_blur_sigma;
+  return exp(-0.5 * x * x / variance) / (kSqrtTwoPi * v_blur_sigma);
```

Comment:
Confirmed that this is the standard gaussian blur equation as long as v_blur_sigma is non-negative.